> 
> 
> `xalan-c` failed on osx pipeline now.
> Could you please take a look and try to fix the regression caused by `xalan-c`?
> [install-x64-osx-dbg-out .log](https://github.com/microsoft/vcpkg/files/4082208/install-x64-osx-dbg-out.log)

How is it working on master?  Unless there is a cached download?  xerces-c cannot be downloaded from the URL used:

```
-- Downloading https://github.com/apache/xerces-c/archive/Xerces-C_3_2_2.tar.gz... Failed. Status: 22;"HTTP response code said error"
CMake Error at scripts/cmake/vcpkg_download_distfile.cmake:175 (message):

      Failed to download file.
```

Because xerces-c has broken on master for a while, we can't determine if this failure on Apple macOS is new to this PR (shouldn't be).

--- a/ports/xerces-c/CONTROL
+++ b/ports/xerces-c/CONTROL
@@ -1,5 +1,5 @@
 Source: xerces-c
-Version: 3.2.2-12
+Version: 3.2.2-13
 Homepage: https://github.com/apache/xerces-c
 Description: Xerces-C++ is a XML parser, for parsing, generating, manipulating, and validating XML documents using the DOM, SAX, and SAX2 APIs.
 

--- a/ports/xerces-c/portfile.cmake
+++ b/ports/xerces-c/portfile.cmake
@@ -1,9 +1,9 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO apache/xerces-c
-    REF 71cc0e831b522933ae58253fcaf4423a166e6bec #3.2.2
-    SHA512 5ddaf84552cd1f685068352c12651780a9f4becd961d8efd0f1f6a21006df00ffb1afe9b9dd86a169f8c57c4f8b651d5477e1accfd7140a7a52aea14b23c9aed
-    HEAD_REF trunk
+    REF v3.2.2
+    SHA512 e4b2d3499fb4d1d1bcaf991ee858f352112683084b9cc7855c0e52e7fc8cc982a8e3cd548fa30718af6a6dee40e460d82ffcd3480a50f7e7516b462b2c2080c6
+    HEAD_REF master
     PATCHES
         disable-tests.patch
         remove-dll-export-macro.patch

